src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -19,52 +19,62 @@ const PanelSideBarItem = (props: PanelSideBarItemProps) => {
   const { depth = 0, children: item, LinkRenderer, onClick, toggledItemIds = [], toggledSidebar } = props;
 
   const hasitem = !!item.children?.length;
+  const isActive = item.children?.find((s) => s.active) || item.active;
   const [isOpen, setIsOpen] = useState(toggledItemIds?.includes(item.id) || item.expanded);
   if (item.display === false) {
     return null;
   }
+  const scrollToActiveItemRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
```

Comment:
@severinoAmmirati This use Effect is dangerous. When you remove the array dependecies it will be executed at every effect of the page. The dependency array should be empty, but still in there

Reply:
@manni497 Restored